core/src/main/java/it/pagopa/selfcare/mscore/core/DelegationServiceImpl.java

```diff
@@ -41,18 +40,15 @@ public DelegationServiceImpl(DelegationConnector delegationConnector,
 
     @Override
     public Delegation createDelegation(Delegation delegation) {
+        if(PROD_PAGOPA.equals(delegation.getProductId())) {
```

Comment:
you could add some comments to explain why this behaviour

Reply:
Done